src/containers/AppError/index.js

```diff
@@ -26,12 +26,19 @@ let AppError = ({ goBack }) => (
     </div>
 
     <div>
-      <img src={SpilledSample} alt="Something went wrong" />
+      <img
+        src={SpilledSample}
+        alt="Something went wrong"
+        className="img-responsive"
+      />
     </div>
   </div>
 );
-AppError = connect(() => ({}), {
-  goBack
-})(AppError);
+AppError = connect(
+  () => ({}),
```

Comment:
This arrow function `() => ({})` is not doing anything, can we simply use `null`?

Reply:
good idea! I wasn't sure of this, but double checked that it's possible in the [connect docs](https://github.com/reduxjs/react-redux/blob/master/docs/api.md#arguments)